### PR TITLE
CNV-29940: Release note Preview bridge NIC hot-plug

### PR DESCRIPTION
**Pull request by aspauldi**
Version(s): 4.14

Issue: [CNV-29940](https://issues.redhat.com/browse/CNV-29940)

Link to docs preview: [Technology Preview features](https://63953--docspreview.netlify.app/openshift-enterprise/latest/virt/release_notes/virt-4-14-release-notes#virt-4-14-technology-preview)

QE review:
- [x] QE has approved this change.

### CODE REVIEW

File: virt/release_notes/virt-4-14-release-notes.adoc
Comment (pcbailey) [inline virt/release_notes/virt-4-14-release-notes.adoc:167]: "Hot-plug" can be used as a verb. "You can hot-plug a bridged network interface to a ..."

There are conditions under which the live migration step isn't necessary: https://kubevirt.io/user-guide/operations/hotplug_interfaces/ (Requirements section). 
I'm not sure if you omitted that on purpose, but I wanted to make sure you're aware in case it needs to be added.

To my knowledge, cluster admin permissions are only necessary to enable the feature.

**Comment by aspauldi** [comment]
Hi @pcbailey, This update includes a rewrite of the text (with input from Avital). Can you please take another look? Thanks!

**Comment by pcbailey** [comment]
@aspauldi It looks good to me!

**Comment by aspauldi** [comment]
@gouyang@redhat.com, Can you please review this PR? Thanks!

File: virt/release_notes/virt-4-14-release-notes.adoc
Comment (xenolinux) [inline virt/release_notes/virt-4-14-release-notes.adoc:167]: Should "hot-plug" be "hot plug"? I was observing other instances in the repo where "hot plug" is used as a verb. For eg: https://docs.openshift.com/container-platform/4.13/virt/virtual_machines/virtual_disks/virt-hot-plugging-virtual-disks.html#virt-about-hot-plugging-virtual-disks_virt-hot-plugging-virtual-disks
> When you hot plug a virtual disk, you attach......
> After you hot plug a virtual disk, it remains.......

**Review by xenolinux — COMMENTED** [review]
I left one suggestion but otherwise looks good!

I am new on the peer review squad. Please wait until @abhatt-rh gives my review the go-ahead before implementing any feedback. Thanks!

**Comment by aspauldi** [comment]
Peer review comment incorporated. Awaiting review by https://github.com/abhatt-rh

**Comment by abhatt-rh** [comment]
> I left one suggestion but otherwise looks good!
> 
> I am new on the peer review squad. Please wait until @abhatt-rh gives my review the go-ahead before implementing any feedback. Thanks!

@aspauldi and @xenolinux: Peer review LGTM!

**Review by jeana-redhat — APPROVED** [review]
/remove-label merge-review-in-progress
/remove-label merge-review-needed

**Pull request merged**
